RIOT-OS/RIOT
# Opt skald advitvlconfig

### Contribution description
Until now, `skald` was implemented in a way that all active advertisings were using the same, compile-time-configured, advertising interval. But making the interval configurable per context make `skald` much more flexible, trading 4 byte in RAM overhead and a single pointer deref - quite OK I'd say :-) The actual change consists of two lines, adding the `adv_itvl_ms` field to the context struct and substituting `CONFIG_SKALD_INTERVAL_MS` for `ctx->adv_itvl_ms` in the implementation. All the rest is just the adaption of the two ibeaon and eddystone wrappers...

### Testing procedure
Run the two provided `skald` examples (`examples/skald_ibeacon`) and (`examples/skald_eddystone`) and verify they still advertise as expected.

### Issues/PRs references
none
